components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link'
+import { HeartIcon, ShoppingBagIcon, UserCircleIcon } from '@heroicons/react/outline'
+
+const Navbar = () => {
+  return (
+    <div className="flex flex-row flex-wrap items-center justify-between py-3 px-5">
+      <div className="order-2 flex flex-row items-center space-x-4 py-1 md:order-1">
+        <Link href="/commerce">
```

Comment:
Still very unclear why there are two links going to the same exact place.

Reply:
Addedd link to github